src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/ApplicationTest.kt

```diff
@@ -928,7 +928,8 @@ class ApplicationTest : IntegrationTestBase() {
     }
   }
 
-  inner class Cas3GetApplication {
+  @Nested
+  inner class Cas3Get {
```

Comment:
Is there a reason to change this class name?

Reply:
This class name results in class filenames which result in errors during build since the filenames are too long.  Reducing the class name length solves this issue.  Added more information to the PR description too.

Reply:
This can be solve by reduce the tests description not the class name which is not make sense

Reply:
See comment above.